parameters/create_parameters.jl

```diff
@@ -161,5 +167,5 @@ function create_climaatmos_parameter_set(
     )
     # logfilepath = joinpath(@__DIR__, "logfilepath_$FT.toml")
     # CP.log_parameter_information(toml_dict, logfilepath)
-    return param_set
+    return param_set, config_params
```

Comment:
I like the split between config params and parameters that need to be passed around in the model. Maybe it would be good to add a docstring somewhere that explains what that function is doing? 

We still have the `examples/hybrid/parameter_set.jl` file with `create_parameter_set`, `baro_wave_parameter_set` and `tc_parameter_set`. Could you also bring some sanity there? Maybe just overriding things as done for the `is_column_without_edmf(parsed_args)` case without specifying the functions for it?